chameleon/settings.py

```diff
@@ -257,6 +257,9 @@
             "PORT": os.environ.get("DB_PORT"),
             "USER": os.environ.get("DB_USER"),
             "PASSWORD": os.environ.get("DB_PASSWORD"),
+            "TEST": {
```

Comment:
Is this built into the django database connection spec or something? I can't find reference to it.

Reply:
Yes, this is mentioned in [django docs](https://docs.djangoproject.com/en/4.2/topics/testing/overview/)

> The default test database names are created by prepending test_ to the value of each [NAME](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-NAME) in [DATABASES](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-DATABASES). When using SQLite, the tests will use an in-memory database by default (i.e., the database will be created in memory, bypassing the filesystem entirely!). The [TEST](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-DATABASE-TEST) dictionary in [DATABASES](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-DATABASES) offers a number of settings to configure your test database. For example, if you want to use a different database name, specify [NAME](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-TEST_NAME) in the [TEST](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-DATABASE-TEST) dictionary for any given database in [DATABASES](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-DATABASES).